Bottom: 22c94c5aec256f8b0e61c22e679f36c33fce9b89
Top:    324dca7a9df2b8ab8a6b959820c1e86638cb8b4e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:34:58 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
